README.md

```diff
@@ -1,8 +1,8 @@
 # Project Name
 
--Portfolio: Project Deploy
+-Portfolio: Mobile Menu
 
-https://telanet.github.io/Portfolio/
+![screenshot](images/screenshot-1.png)
 
 # Getting Started
 -To get a local copy up and running follow these simple example steps.
```

Comment:
- [ ] Please add more procedures to clone and run your project locally in this section. This section is more important for some your fellow developers who are interested in your project 😉 🏃